app/setup.py

```diff
@@ -323,21 +337,23 @@ def setup_redis(application):
 
 def setup_submitter(application):
     if application.config["EQ_SUBMISSION_BACKEND"] == "gcs":
-        bucket_name = application.config.get("EQ_GCS_SUBMISSION_BUCKET_ID")
-
-        if not bucket_name:
-            raise Exception("Setting EQ_GCS_SUBMISSION_BUCKET_ID Missing")
-
-        application.eq["submitter"] = GCSSubmitter(bucket_name=bucket_name)
+        if bucket_name := application.config.get("EQ_GCS_SUBMISSION_BUCKET_ID"):
+            application.eq["submitter"] = GCSSubmitter(bucket_name=bucket_name)
+        else:
+            raise MissingEnvironmentVariable(
+                "Setting EQ_GCS_SUBMISSION_BUCKET_ID Missing"
+            )
```

Comment:
Minor, but I think the previous way is clearer. We should escape early to reduce indentation, this changes increases the indentation. You can still exist early and make use of walrus.

Reply:
Changed back to the old version but refactored to use walrus operator 👍